spatial-tracking-explainer.md

```diff
@@ -364,7 +391,9 @@ dictionary XRReferenceSpaceOptions {
   required XRReferenceSpaceType type;
 };
 
-[SecureContext, Exposed=Window] interface XRReferenceSpace : XRSpace {
+[SecureContext, Exposed=Window] interface XRReferenceSpace : XRSpace {  
+  attribute XRRigidTransform? baseTransform;
```

Comment:
I can't recall who suggested offset, but I liked that too.  `originOffset` 

I do not see offset as implying translation only, but I can see how some might, though.

I'm not a fan of "base".

Or, `originOffset`?

Reply:
I suspect you intended to not suggest `originOffset` twice, so now I'm curious what the other suggestion was supposed to be? 😉 

In general, though, I'm seeing a lot of support for the term `origin` and more support than expected for some variant of `offset`, which likely means that my concern about it's implied meaning is overblown. Going to move for the moment to `originTransform` since it gets away from the `base` term that doesn't seem too popular and I've now heard it suggested from a couple of different sources (thanks @darktears!) but I could be persuaded to go with `originOffset` if that has more support.

Reply:
As it was highlighted in IRC yesterday (and thus my suggestion in the first place) origin is consistent with the web platform, see https://developer.mozilla.org/en-US/docs/Web/CSS/transform-origin

Reply:
I missed that in IRC yesterday, thanks for repeating it here! I definitely think consistency with the rest of the web platform is a positive thing!

Reply:
I do somewhat prefer `originOffset` to `originTransform`, as it more strongly suggests that there is a natural origin that is meaningful in some way, but which you can offset relative to that natural location if you need to, e.g. to accomplish teleportation.

Most other attributes in the API named `transform` or `...Transform` can be though of fairly simply as "the absolute position and orientation of the thing within the space", which is not really what this relative offset is, and so perhaps it's good that the attribute name is a little different.